### ISSUE TITLE
Further automatization

### ISSUE THREAD

**Issue by Aqa-Ib** [Open-Wine-Components/umu-protonfixes#29]
Maybe we could take advantage of the fact that Steam already contains protonfixes. 

Are their protonfixes closed source ?

**Comment by GloriousEggroll** [comment]
I'm not sure what you mean by this. Valve/Steam official does -not- include any verson of protonfixes or winetricks or anything like that.

**Comment by Aqa-Ib** [comment]
I mean the fixes themselves. For example, when you install Grim Dawn from Steam, it will install the necessary d3dcompiler_43 automatically. How ? Does it just install all windows depots listed on steamdb? or does it pull the fix from somewhere else ?

**Comment by GloriousEggroll** [comment]
> I mean the fixes themselves. For example, when you install Grim Dawn from Steam, it will install the necessary d3dcompiler_43 automatically. How ? Does it just install all windows depots listed on steamdb? or does it pull the fix from somewhere else ?

It doesn't man. We do that. Proton-GE and ULWGL-Proton insert protonfixes into valve's proton here:

https://github.com/GloriousEggroll/proton-ge-custom/blob/ad76ca13583c3aa9c27d91104dd8b1fe94362bb7/proton#L1565

Which then triggers the game fixes we've added:

https://github.com/Open-Wine-Components/ULWGL-protonfixes/blob/533777de4fa998d1e4490ef968e03fefa7233ea6/gamefixes-steam/219990.py#L18

**Comment by Aqa-Ib** [comment]
I mean, the game is listed in Steam as playable on the Steam deck. You buy the game on Steam and it works out of the box without having to install anything. Am I mistaken ?

**Comment by GloriousEggroll** [comment]
> Let me explain myself, please. The game is listed in Steam as playable on the Steam Deck which apparently means that you buy the game on Steam and it works out of the box without having to install anything else. Am I mistaken ?

All that means is that the fix we added either isn't needed any more or was added at some point for something such as an addon or mod, or it possibly fixes some gameplay related bug. Grim Dawn has been in our protonfixes list for a long time. It's very likely d3dcompiler_43 may not be needed any more or possibly fixes some in-game graphic issue or crash unrelated to launching the game.
